preprocessors/openstreetmap/main.py

```diff
@@ -54,26 +65,26 @@ def get_map_data():
 
     if validated is not None:
         return validated
-
+    time_stamp = int(get_timestamp())
+    name = "ca.mcgill.a11y.image.preprocessor.openstreetmap"
+    request_uuid = content["request_uuid"]
     # Check if this request is for an openstreetmap
     if 'coordinates' not in content and 'placeID' not in content:
-        logging.info("Not map content. Skipping...")
-        return "", 204
+        LOGGER.info("Not map content. Skipping...")
+        return jsonify(""), 204
 
     # Build OpenStreetMap request
     coords = get_coordinates(content)
     if coords is None:
-        error = 'Unable to find Latitude/Longitude'
-        logging.error(error)
-        return jsonify(error), 400
+        error = 'Unable to find API key'
```

Comment:
I am not a fan of this error message. This does not distinguish between the API key not being loaded and an error in getting a response from Google. For example, if we request an invalid place ID, this error will still occur and mislead the debugger.